compiler/qsc_qasm3/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "qsc_qasm3"
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+edition.workspace = true
+license.workspace = true
+version.workspace = true
+
+[dependencies]
+bitflags = { workspace = true }
+num-bigint = { workspace = true }
+miette = { workspace = true }
+qsc = { path = "../qsc" }
+rustc-hash = { workspace = true }
+thiserror = { workspace = true }
+oq3_source_file = { workspace = true }
+oq3_syntax = { workspace = true }
+oq3_parser = { workspace = true }
+oq3_lexer = { workspace = true }
+oq3_semantics = { workspace = true }
+
+[dev-dependencies]
+difference = { workspace = true }
+expect-test = { workspace = true }
+indoc = { workspace = true }
+miette = { workspace = true, features = ["fancy"] }
+qsc_qasm3 = { path = ".", features = ["fs"] }
```

Comment:
Is the self-dev dependency here just to make sure the `fs` feature is enabled? This could be worth a comment if so, so that folks know why this shouldn't be removed.